src/routes/safe/components/Apps/communicator.ts

```diff
@@ -66,8 +68,8 @@ class AppCommunicator {
           this.send(response, msg.data.id)
         }
       } catch (err) {
-        console.log({ err })
         this.send(err.message, msg.data.id, true)
+        logError(Errors._602, `${msg.data.method} ${err.message}`)
```

Comment:
Be advised that this will also send the error to Sentry which we previously didn't do. I think it's a good move.
If this is not desired though, there's a third boolean param to only log to the console.